basyx.aasenvironment/basyx.aasenvironment-core/src/main/java/org/eclipse/digitaltwin/basyx/aasenvironment/AasEnvironmentSerialization.java

```diff
@@ -42,12 +42,13 @@ public interface AasEnvironmentSerialization {
 	 * 
 	 * @param aasIds
 	 * @param submodelIds
+	 * @param includeConceptDescriptions
 	 * @return
 	 * @throws SerializationException
 	 */
-	public String createJSONAASEnvironmentSerialization(List<String> aasIds, List<String> submodelIds) throws SerializationException;
+	public String createJSONAASEnvironmentSerialization(List<String> aasIds, List<String> submodelIds, Boolean includeConceptDescriptions) throws SerializationException;
```

Comment:
Why are you using Boolean instead of boolean here?

Reply:
Replaced the Boolean wrapper class with the primitive